pom.xml

```diff
@@ -161,7 +162,35 @@
     </dependencyManagement>
 
     <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>com.github.cafapi.plugins.docker.versions</groupId>
+                    <artifactId>docker-versions-maven-plugin</artifactId>
+                    <version>1.0.0-SNAPSHOT</version>
+                </plugin>
+            </plugins>
+        </pluginManagement>
         <plugins>
+            <plugin>
+                <groupId>com.github.cafapi.plugins.docker.versions</groupId>
+                <artifactId>docker-versions-maven-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <imageManagement>
+                        <image>
+                            <repository>${dockerHubPublic}/cafapi/opensuse-jre11</repository>
+                            <tag>4.0.0</tag>
+                            <digest>sha256:03c2b7fe78f3db1831a492ca341266eca7efb92f1f4968939308b9d038aba97c</digest>
+                        </image>
+                        <image>
+                            <repository>${dockerHubPublic}/opensuse/leap</repository>
+                            <tag>15.5</tag>
+                            <digest>sha256:da3a32575d87b0f01b8db23d481db00ede78bf7dcab57b1c653aa1d8255068c6</digest>
```

Comment:
I'm reasonably certain that this isn't a fixed tag.

Reply:
I looked at https://hub.docker.com/r/opensuse/leap/tags?page=&page_size=&name=&ordering=name and used the docker-versions:use-latest-releases and 15.5 comes up as the latest 'longest' tag.

Reply:
I think it's likely that openSUSE don't publish fixed tags at all.